src/__test__/utils/builders.ts

```diff
@@ -76,30 +79,48 @@ export const buildFirmwareConstants = (...overrides: any) => {
   } as FirmwareConstants;
 };
 
-export const buildGetAddressesObject = ({ ...overrides }) => ({
+export const buildWallet = (overrides?) => ({
+  uid: Buffer.from(
+    '162b56efe561c12bc93f703dc7026b3ec3d53923270c9259e2b08015fb9defd2',
+    'hex',
+  ),
+  name: Buffer.from('test'),
+  capabilities: 1,
+  external: true,
+  ...overrides,
+});
+
+export const buildGetAddressesObject = (overrides?) => ({
   startPath: [0x80000000 + 44, 0x80000000 + 60, 0x80000000, 0, 0],
   n: 1,
   flag: 1,
-  client: buildMockConnectedClient(),
+  fwConstants: buildFirmwareConstants(),
+  url: 'asdf',
```

Comment:
How does this work? Does the url just never get checked by anything that uses this function? Seems like a client validator should check for `http` prefix

Reply:
I removed it, since it isn't required for any tests that use it. 

But I also added better validation for `url` and `baseUrl`.